src/no-random-v4-uuid.spec.ts

```diff
@@ -70,14 +80,95 @@ createTester().run(ruleId, rule, {
                const uuidV4 = v4();
             
                const uuidV5 = v5('name', namespace);`,
+      errors: [
+        {
+          messageId: 'NO_UUID_MODULE_FOR_V4',
+        },
+        {
+          messageId: 'NO_UUID_MODULE_FOR_V4',
+        },
+      ],
+    },
+    {
+      name: 'Invalid case with crypto.randomUUID',
+      code: `import { randomUUID } from 'node:crypto';
+         const id = randomUUID();`,
       errors: [
         {
           messageId: 'NO_RANDOM_V4_UUID',
         },
+      ],
+    },
+    {
+      name: 'Invalid case with alias import from crypto',
+      code: `import { randomUUID as uuid } from 'node:crypto';
+         const id = uuid();`,
+      errors: [
         {
           messageId: 'NO_RANDOM_V4_UUID',
         },
       ],
     },
+    {
+      name: 'Invalid case with all the named exports crypto namespace',
+      code: `import * as crypto from 'node:crypto';
```

Comment:
all these tests are the `node:crypto` module, but using the standard JS `crypto` built-in should also be tested.